docs/Dev_eXo_Addons.rst

```diff
@@ -1139,11 +1142,87 @@ following:
         </component-plugin>
       </external-component-plugins>  
       
-     
+
+
+.. _PLFDevGuide.eXoAdd-ons.RewardPlugin:
+
+==============================
+eXo Reward plugin
+==============================
+
+eXo Reward package is a set of addons used to reward employees of an enterprise of their professional effort.
+The list of addons included are:
+
+* `eXo Wallet <https://github.com/exoplatform/wallet>`__
+
+* `eXo Kudos <https://github.com/exoplatform/kudos>`__
+
+* `eXo Perk Store <https://github.com/exoplatform/perk-store>`__
+
+Built-in reward plugins are:
+
+* `Kudos reward plugin <https://github.com/exoplatform/wallet/blob/develop/wallet-reward-services/src/main/java/org/exoplatform/addon/wallet/reward/plugin/KudosRewardPlugin.java>`__
+
+* `Gamification reward plugin <https://github.com/exoplatform/wallet/blob/develop/wallet-reward-services/src/main/java/org/exoplatform/addon/wallet/reward/plugin/GamificationRewardPlugin.java>`__
+
+A reward plugin consist of collecting earned `points` for each user during a period of time.
+When the Rewards package administrator will pay rewards to users for a period of time (each month for example),
+the rewards plugins will be used to retrieve earned points for each user having a wallet.
+The retrieved earned points will be used to compute amount of tokens to send to each user.
+
+By example, the `Kudos reward plugin` is used to return the number of Kudos received by a user during a selected period of time.
+
+In order to define a new Reward plugin, we will need:
```

Comment:
I suggest we could have a more compelling example instead of a void custom plugin. For example, how about providing an even bounty plugin, that simply rewards users evenly based on budget ? (everybody gets the same amount of tokens from this plugin in the end).

Reply:
ok, done